services/core-web/common/components/documents/ArchivedDocumentsSection.tsx

```diff
@@ -8,7 +8,7 @@ import { ColumnType } from "antd/es/table";
 import { useFeatureFlag } from "@mds/common/providers/featureFlags/useFeatureFlag";
 
 interface ArchivedDocumentsSectionProps {
-  documents: MineDocument[];
+  documents?: MineDocument[];
```

Comment:
I do find it odd that the props were changed for the core-web/MS versions but not the common one.